src/StaticPages/SponsorshipPage.tsx

```diff
@@ -0,0 +1,102 @@
+import { format, addMonths, setDate } from 'date-fns'
+import { range } from 'lodash'
+import { Link, Page } from 'src/components'
+import { sponsorConferenceForDate } from 'src/components/SponsoredConference/utils'
+
+import styles from './SponsorshipPages.module.scss'
+
+// Set date to middle of the month to prevent from potential time bug
+const TODAY = setDate(new Date(), 15)
+
+// Create an array of the next 10 months with availability
+// Returns:
+//    [
+//      ['September 2022', false]
+//      ['October 2022', true]
+//      ...
+//    ]
+const SPONSORS_AVAILABILITY = range(0, 10).map((index) => {
+  const sponsorDate = addMonths(TODAY, index)
+  const isMonthTaken = sponsorConferenceForDate(sponsorDate)
+  return [format(sponsorDate, 'MMMM yyyy'), !isMonthTaken]
+})
+
+export default function SponsorshipPage() {
+  return (
+    <Page
+      narrow
+      title='Sponsorship'
+      htmlTitle='Become a sponsor of Confs.tech'
+      backButton
+    >
+      <p>
+        We reach an average of 15,000 unique visitors per months, and our
+        current sponsorship rate is 100€ per month.
+        <br />
+        Your conference will be sponsored at the top of the list for the
+        duration of the sponsorship. We encourage event organizers to also give
+        a special discount for sponsored events.
```

Comment:
Maybe it should be 'promoted' instead of 'sponsored'?

Reply:
Ah that's a good point. I guess my idea behind "sponsorship" is that there's the sens of _supporting_ Confs.tech, not just promoting their event. But I think you're right though... 🤔 

I'll update the wording!

Reply:
I think the other reason I went with sponsorship instead of promotion, is because most creators talk about sponsorship and not promotion. 🤔

Reply:
For me it sounded like we are sponsoring them and not the other way around :)

Also here: `Your conference will be sponsored at the top of the list`.

Could be `Your conference will be shown at the top of the list`
or
`When sponsoring Confs.tech, your conference will be shown at the top of the list for the duration of the sponsorship. We encourage event organizers to also give a special discount for the promoted events.`

Reply:
Ah, good call, I'll update that! 👍🏼